sagemaker_neo_compilation_jobs/imageclassification_caltech/Image-classification-fulltraining-highlevel-neo.ipynb

```diff
@@ -7,9 +7,10 @@
     "# Model Optimization with an Image Classification Example\n",
```

Comment:
<p>a typo:</p><p><code>image_uri</code>: This is set to the training_image uri we defined previously. Once set, this image will be used later while <strong>runnnig</strong><u> </u>the training job.</p><p>to</p><p><code>image_uri</code>: This is set to the training_image uri we defined previously. Once set, this image will be used later while <strong>running</strong> the training job.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
done